tools/verify-golint.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
```

Comment:
Is this copied from somewhere else? Which pieces have changed?

Reply:
Updated the commit message